diff --git a/index.js b/index.js
index bef15fc1ccfe989d8479aa08e63ce49a82e74ef0..0a89a7e7daff09c698cc32e78111a9d300a971c3 100644
--- a/index.js
+++ b/index.js
@@ -12,6 +12,9 @@ const DEFAULT_FUNCTION_NAME = 'fn';
 
 const get_default_runtime = () => {
 	const major = process.version.slice(1).split('.')[0];
+	console.log({
+		'node-version': process.version
+	})
 	if (major === '18') return 'nodejs18.x';
 	if (major === '20') return 'nodejs20.x';
 
